filter_plugins/cisco_nxos.py

```diff
@@ -0,0 +1,443 @@
+import re
+
+def getVLANList(config):
```

Comment:
I don't know cisco config syntax. Should `getVLANList("vlan 1")` return `["1"]` or `[]`?

Reply:
@hakasapl wanted to draw your attention to this; is the current code wrong, or will we never see a `vlan` line without commas?

Reply:
Even if there isn't a comma shouldn't split() then just output a list of size 1 and work anyway? In the latest commit I moved the default vlan stuff around a bit to make more sense. That's added in the main filter plugin method though